include/switcher.php

```diff
@@ -55,22 +57,31 @@ protected function get_elements( $links, $args ) {
 			$locale = $language->get_locale( 'display' );
 			$classes = array( 'lang-item', 'lang-item-' . $id, 'lang-item-' . esc_attr( $slug ) );
 			$url = null; // Avoids potential notice
-
-			if ( $current_lang = $links->curlang->slug == $slug ) {
-				if ( $args['hide_current'] && ! ( $args['dropdown'] && ! $args['raw'] ) ) {
-					continue; // Hide current language except for dropdown
-				} else {
-					$classes[] = 'current-lang';
+			$current_lang = ''; // Avoids potential notice
+
+			if ( 0 === $args['admin_render'] ) {
+				if ( $current_lang = $links->curlang->slug == $slug ) {
+					if ( $args['hide_current'] && ! ( $args['dropdown'] && ! $args['raw'] ) ) {
+						continue; // Hide current language except for dropdown
+					} else {
+						$classes[] = 'current-lang';
+					}
 				}
```

Comment:
I don't understand, wasn't that code supposed to be executed outside of a language context? Why checking on the current language here?

Reply:
Just because of the language switcher feature: option 'Hides the current language'